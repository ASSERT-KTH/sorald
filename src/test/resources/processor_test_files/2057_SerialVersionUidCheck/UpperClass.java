import java.io.Serializable;

class UpperClass implements Serializable {
}
