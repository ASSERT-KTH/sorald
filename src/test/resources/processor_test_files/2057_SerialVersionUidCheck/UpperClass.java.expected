// Test for rule s2057
import java.io.Serializable;

class UpperClass implements Serializable { // Noncompliant
  static final long serialVersionUID = 1L;
}
